src/styles.scss

```diff
@@ -77,6 +77,50 @@ $carbon--theme: $carbon--theme--g10;
 @import 'globals/grid';
 @import 'globals/charts';
 
+// Font families needed for Charts (not included by carbon by default)
+@font-face {
+  font-family: 'IBM Plex Sans Condensed';
+  font-style: italic;
+  font-weight: 300;
+  src: url(~@ibm/plex/IBM-Plex-Sans-Condensed/fonts/complete/woff2/IBMPlexSansCondensed-LightItalic.woff2)
```

Comment:
If we're going to self host the plex files we should update all weights to use the self hosted version to avoid version mismatching between the default weights provided by `@carbon/type` and these additional weights. 

Also, this is going to increase our bundle size by a decent amount. What about [using unpkg instead](https://github.com/carbon-design-system/carbon/blob/master/docs/guides/ibm-plex.md#unpkg)?

Reply:
I worry that we're forcing a bloated bundle on consumers by adding plex as self hosted. 

unpkg is a good option, as is google fonts. Either of these allow consumers to decide how best to load the fonts (their own CDN or self host), with the google fonts fallback/default that falls in line with `@carbon/type`

Reply:
```diff
- src: url(~@ibm/plex/IBM-Plex-Sans-Condensed/fonts/complete/woff2/IBMPlexSansCondensed-LightItalic.woff2)
+ src: url(//unpkg.com/@ibm/plex/IBM-Plex-Sans-Condensed/fonts/complete/woff2/IBMPlexSansCondensed-LightItalic.woff2)
```